plugins/bigquery/dbt/include/bigquery/macros/adapters.sql

```diff
@@ -36,24 +36,25 @@
   {% if temporary %}
     {% do opts.update({'expiration_timestamp': 'TIMESTAMP_ADD(CURRENT_TIMESTAMP(), INTERVAL 12 hour)'}) %}
   {% endif %}
+  {% if kms_key_name %}
+    {% do opts.update({'kms_key_name': "'" ~ kms_key_name ~ "'"}) %}
+  {% endif %}
 
-  {% set options -%}
-    OPTIONS({% for opt_key, opt_val in opts.items() %}
-      {{ opt_key }}={{ opt_val }}{{ "," if not loop.last }}
-    {% endfor %})
-  {%- endset %}
-  {% do return(options) %}
+  OPTIONS({% for opt_key, opt_val in opts.items() %}
```

Comment:
I think we want to keep some of the logic that was overwritten here. It looks to me like you may not have pulled `dev/louisa-may-alcott` before making this PR? I think this code was _just_ merged a couple of days ago.

The `{% set %}` block is used to better manage whitespace in the compiled SQL output. The `{{ "," if not loop.last }}` construct is used to ensure that these options are comma-delimited.

Reply:
I was definitely behind. Thanks for catching!